admin/templates/spam/email.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}
+    <title>Email</title>
+{% endblock title %}
+{# stylesheets #}
+{% block content %}
+    {{ block.super }}
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="btn-group col-sm-12">
+                <a href="{% url 'spam:spam' %}" class="btn btn-primary">
+                    Back to list
+                </a>
+                <a href="{% url 'spam:detail' comment.id %}"
```

Comment:
```html
<a href="javascript:window.history.back();return true;">Back to list</a>
```
will work and it's not *too* awful. The only other thing I could think of is to pass the current page number to all the detail links in a query string and then build the back to list links using that. But the javascript way is quicker and less fragile. This was more a usability thing. I created 3500 comments and 4500 spam reports and then started poking around in the admin. It was inconvenient to go from page 35 to detail to page 1.